acq4/analysis/dataModels/PatchEPhys/GetClamps.py

```diff
@@ -0,0 +1,331 @@
+# -*- coding: utf-8 -*-
+"""
+
+This is part of Acq4
+
+Paul B. Manis, Ph.D.
+2011-2013.
+
+Pep8 compliant (via pep8.py) 10/25/2013
+Refactoring begun 3/21/2015
+
+"""
+
+
+import os
+import os.path
+import numpy as np
+import re
+from acq4.analysis.AnalysisModule import AnalysisModule
+from acq4.util.metaarray import MetaArray
+
+
+# noinspection PyPep8
+class GetClamps(AnalysisModule):
```

Comment:
Or maybe cell_summary can be put in datamodel/patchephys, and the class can go away.